plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/ParquetTester.java

```diff
@@ -192,7 +192,7 @@ public ParquetTester(
         this.fileFormat = requireNonNull(fileFormat, "fileFormat is null");
     }
 
-    public void testRoundTrip(PrimitiveObjectInspector columnObjectInspector, Iterable<?> writeValues, Type parameterType)
+    public <T> void testRoundTrip(PrimitiveObjectInspector columnObjectInspector, Iterable<T> writeValues, Type parameterType)
```

Comment:
unrelated. Extract commit.

Reply:
It is related to the refactoring (compiler was failing at some call sites). Extracting it into a separate commit might make it easier to review, but then it would be hard to understand why this chance was made by looking solely on the commit that introduces it.